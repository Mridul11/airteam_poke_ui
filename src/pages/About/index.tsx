import './about.css';

export default function About() {
  return (
    <div className="about-center">About</div>
  )
}
