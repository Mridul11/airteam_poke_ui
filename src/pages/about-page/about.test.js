import { shallow } from 'enzyme';
import AddWatchMedia from '../../utils/test-utils/add-matchmedia';
import About from '.';

describe('App Component', () => {
  let wrapper;
  beforeAll(AddWatchMedia);
  beforeEach(() => (wrapper = shallow(<About />)));
  it('should render correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
