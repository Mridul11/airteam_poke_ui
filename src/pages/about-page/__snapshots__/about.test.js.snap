// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About Component should match snapshot 1`] = `
<div
  className="about-center"
  data-testid="test-aboutpage"
>
  <Spinner />
</div>
`;
